### PR TITLE
Splits Apart kind-image-fast Make Target

### PR DESCRIPTION
**Pull request by danehans**
Splits apart the `kind-image-fast` Make target to allow for targeted test/dev workflows.

### CODE REVIEW

**Comment by danehans** [comment]
cc: @aanm

File: Makefile [hunk 1]
```diff
@@ -604,20 +615,43 @@ kind-image-fast: ## Build cilium, operator, clustermesh-apiserver and mount them
...
 			docker exec -ti $${node_name} chmod +x "${dst}/clustermesh-apiserver"; \
 			\
-			kubectl --context=kind-$${cluster_name} delete pods -n kube-system -l k8s-app=cilium --force; \
-			kubectl --context=kind-$${cluster_name} delete pods -n kube-system -l name=cilium-operator --force; \
 			kubectl --context=kind-$${cluster_name} delete pods -n kube-system -l k8s-app=clustermesh-apiserver --force; \
 		done; \
 	done
 
+.PHONY: kind-image-fast
+kind-image-fast: kind-image-fast-agent kind-image-fast-operator kind-image-fast-clustermesh-apiserver ## Build all binaris and copy them to kind nodes.
```
Comment (aanm) [inline Makefile]: s/binaris/binaries

**Review by aanm — APPROVED** [review]

File: Makefile (hunk 1 above)
Comment (danehans) [reply Makefile]: @aanm thanks for the review. Commit bfd091c resolves this comment.

**Comment by danehans** [comment]
The `Cilium IPsec upgrade` job appears to be failing due to https://github.com/cilium/cilium/issues/28088.

**Comment by danehans** [comment]
@aanm do you have any recommendations for other maintainers to review this PR?

**Pull request merged**